src/tooltip/tooltip.js

```diff
@@ -136,7 +136,7 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
           'class="uib-position-measure ' + prefix + '" ' +
           'tooltip-animation-class="' + animationClass +'"' +
           'uib-tooltip-classes ' +
-          'ng-class="{ ' + openClass +': isOpen }" ' +
+          'ng-class="{ \'' + openClass +'\': isOpen }" ' +
```

Comment:
This change was needed to allow the configurable class name to contain characters like dashes, etc.  Otherwise angular would throw errors since t was not enclosed in a string.